cepheus-cep/README.md

```diff
@@ -1,7 +1,7 @@
 # Cepheus-CEP
 
 Cepheus-CEP is a CEP (Complex Event Processor), it uses the [Esper](http://www.espertech.com/esper/) engine.
-This engine provides a lot of features (aggregation, filters, rate limiting, correlation, partitionning...) with a nice [SQL like syntax](http://www.espertech.com/esper/release-5.2.0/esper-reference/html/epl_clauses.html).
+This engine provides a lot of features (aggregation, filters, rate limiting, correlation, partitionning...) with a nice [SQL like syntax](http://www.espertech.com/release-5.2.0/esper-reference/html_single/index.html#epl_clauses).
```

Comment:
The new link is not working...

This one seems to work: http://esper.espertech.com/release-5.2.0/esper-reference/html/epl_clauses.html

Reply:
I feel sorry that I wrote bad link by mistake.
I replaced it with the link you suggested.
thank you for your advise.